### PR TITLE
[TM ONLY] Hijacked Revision Date for v3 TGS Api

### PR DESCRIPTION
**Pull request by ZephyrTFA**
Adds revision date generation for the v3 TGS Api.
Because of float precision it usually be within +-8 minutes.
This is better than not having it at all and is something I need for CompileMon

### CODE REVIEW

File: code/modules/tgs/v3210/api.dm [hunk 1]
```diff
@@ -66,8 +67,14 @@
 	var/list/logs = TGS_FILE2LIST(".git/logs/HEAD")
 	if(logs.len)
 		logs = splittext(logs[logs.len], " ")
-		if (logs.len >= 2)
+		if (logs.len >= 5)
 			commit = logs[2]
```
Comment (Cyberboss) [inline code/modules/tgs/v3210/api.dm:71]: If >= 5 should be after this line

**Review by Cyberboss — DISMISSED** [review]
Bruh, the bot is gonna stomp this. Just do it in `getrev.dm` if the major version is 3.

**Comment by ZephyrTFA** [comment]
did you not read the TM Only line.
This will never get merged and if someone does I will revert it.
This PR will exist until TG updates TGS

**Comment by optimumtact** [comment]
Figure out a way to make this a compile time define or something I dont want this TM sitting here foreever.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
If this is going to be merged, do away with the TGS internals hack and handle it in getrev.dm https://github.com/tgstation/tgstation/blob/62edc2f93fd25a4c29cb4f8483199b0c82a80df3/code/datums/helper_datums/getrev.dm#L13-L20

If `world.TgsVersion().suite == 3`

**Pull request closed without merging**